e2e/configmap.go

```diff
@@ -68,14 +68,14 @@ func createConfigMap(pluginPath string, c kubernetes.Interface, f *framework.Fra
 	if err == nil {
 		_, updateErr := c.CoreV1().ConfigMaps(cephCSINamespace).Update(context.TODO(), &cm, metav1.UpdateOptions{})
 		if updateErr != nil {
-			return updateErr
+			return fmt.Errorf("failed to update configmap %w", updateErr)
 		}
 	}
 	if apierrs.IsNotFound(err) {
 		_, err = c.CoreV1().ConfigMaps(cephCSINamespace).Create(context.TODO(), &cm, metav1.CreateOptions{})
 	}
 
-	return err
+	return fmt.Errorf("error=%w", err)
```

Comment:
This becomes problematic when `err == nil`

Reply:
True, but it shows wrap-check error. will check the alternative in this case.